pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/SegmentDeletionManager.java

```diff
@@ -54,21 +58,22 @@
   private static final long MAX_DELETION_DELAY_SECONDS = 300L;  // Maximum of 5 minutes back-off to retry the deletion
   private static final long DEFAULT_DELETION_DELAY_SECONDS = 2L;
   private static final String DELETED_SEGMENTS = "Deleted_Segments";
+  private static final String RETENTION_PERIOD_SEPARATOR = "__RETENTION_MS__";
```

Comment:
The suffix is ms, not period
```suggestion
  private static final String RETENTION_MS_SEPARATOR = "__RETENTION_MS__";
```